### DISCUSSION TITLE
Uhh Hey, I Just Want A Question Real Quick,

### DISCUSSION THREAD

**Discussion by eegychv** [character-map-uwp/Character-Map-UWP#232] [Q&A] [answered]
where's the location of Character Map UWP App Folder At C:/Programing Files/WindowsApps?

**Comment by EdiWang** [comment, marked as answer]
`"C:\Program Files\WindowsApps\58027.265370AB8DB33_2023.1.1.0_x64__fjemmk5ta3a5g\CharacterMap.exe"`

**Reply by eegychv** [reply]
Thank You!
